hub/providers/Proposal/createProposal.ts

```diff
@@ -38,7 +38,7 @@ import { NFTWithMeta, VotingClient } from '@utils/uiTypes/VotePlugin';
 import { fetchPlugins } from './fetchPlugins';
 
 interface Args {
-  callbacks?: Parameters<typeof _createProposal>[11];
```

Comment:
you can just ignore this, everything in the hubs folder is sunset

Reply:
done, revert changes from this file

Reply:
re-added changes in the file. This file was giving error in Vercel build if not changed.